ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -607,6 +685,17 @@ wan_virtual_topologies:
             - names: [MPLS]
               preference: preferred
           id: 5
+        - application_profile: CRITICAL-APP
+          path_groups:
+            - names: [INET]
+              preference: preferred
+            - names: [AWS]
+              preference: alternate
+          constraints:
+            loss_rate: 45.0
+          internet_exit:
+            policy: ZSCALER-EXIT-POLICY-3
+          id: 6
```

Comment:
Why do we need a new application profile? It would simplify the test changes a lot if we don't add extra stuff unless strictly needed.